### ISSUE TITLE
[bug] Move Source animation cuts off when running 2 animations

### ISSUE THREAD

**Issue by Venzo92** [exeldro/obs-move-transition#226]
I have made 4 move source animations that position the webcam box to the 4 corners of the screen, inside the filter panel they are ordered as follows;
Top Right
Top Left
Bottom Right
Bottom Left
If i run the top right animation and before it finishes i run any other animation below the behavior is correct as the animation halts middleway and the second animation proceeds to move the box to the right position

if i run bottom left animation and then before it finishes i run any other animation above the behavior is incorrect as the first animation completes normally and then the box gets teleported to the second position

basically the ordering of the animations in the filter list affects the animation outcome
